app/services/catalog/provider_control_parameters.rb

```diff
@@ -7,13 +7,14 @@ def initialize(portfolio_item_id)
     end
 
     def process
-      source_ref = PortfolioItem.find(@portfolio_item_id).service_offering_source_ref
-      TopologicalInventory.call do |api_instance|
-        # TODO: Temporay till we get this call in the topology service
-        projects = api_instance.list_source_container_projects(source_ref).data
-        update_project_list(project_names(projects))
-        self
-      end
+      # source_ref = PortfolioItem.find(@portfolio_item_id).service_offering_source_ref
```

Comment:
@gmcculloug Would the UI have to be changed to handle an empty hash coming back. if it is then we have to make the change in 2 places UI & API. In that scenario should we just have the UI comment out the call to fetch the provider control parameters and it will be in 1 place.

Reply:
@lgalis do you have insight into @mkanoor's question here. Would the UI need to change in line with these changes?

Reply:
@lgalis Ping.  I don't see why we would not do both, fix the UI to handle an empty object and this change until we understand what we need here going forward with other source types.